### ISSUE TITLE
Templates for issues and pull requests

### ISSUE THREAD

**Issue by tarlepp** [makeusabrew/bootbox#670]
We should add simple templates for `issues` and `pull requests` to make some guidelines for community. https://blog.github.com/2016-02-17-issue-and-pull-request-templates/

**Comment by tiesont** [comment]
Agreed. I will add this to the v5 ship list, unless you want it to happen sooner.
